packages/contracts-bedrock/src/l1/SuperchainConfig.sol

```diff
@@ -0,0 +1,122 @@
+    ///         This is expected to be the security council.
+    address public initiator;
+
+    /// @notice The address of the vetoer.
+    ///         This is expected to the foundation.
+    address public vetoer;
+
+    /// @notice The address of the guardian, can pause the OptimismPortal.
+    address public guardian;
+
+    /// @notice The delay time in seconds between when an upgrade is initiated and when it can be finalized.
+    uint256 public delay;
+
+    /// @notice The pause status of withdrawals from an chain in the superchain.
+    bool public paused;
+
+    /// @notice Mapping of allowed sequencers.
+    ///         The initiator should be able to add to it instantly, but removing is subject to delay.
+    mapping(bytes32 => bool) public allowedSequencers;
+
+    /// @notice Emitted when configuration is updated.
+    /// @param version    SystemConfig version.
+    /// @param updateType Type of update.
+    /// @param data       Encoded update data.
+    event ConfigUpdate(uint256 indexed version, UpdateType indexed updateType, bytes data);
+
+    /// @notice Semantic version.
+    /// @custom:semver 1.0.0
+    string public constant version = "1.0.0";
+
+    /// @notice Constructs the SuperchainConfig contract. Cannot set
+    ///         the owner to `address(0)` due to the Ownable contract's
+    ///         implementation, so set it to `address(0xdEaD)`
+    constructor() {
+        initialize({
+            _owner: address(0xdEaD),
+            _initiator: address(0),
+            _vetoer: address(0),
+            _guardian: address(0),
+            _delay: 0,
+            _paused: false,
+            _sequencers: new Types.SequencerKeys[](0)
+        });
+    }
+
+    /// @notice Initializer.
+    ///         The resource config must be set before the require check.
+    /// @param _owner     Initial owner of the contract.
+    /// @param _initiator Address of the initiator who may initiate an upgrade or change to critical config values.
+    /// @param _vetoer    Address of the vetoer.
+    /// @param _guardian  Address of the guardian, can pause the OptimismPortal.
+    /// @param _delay     The delay time in seconds between when an upgrade is initiated and when it can be finalized.
+    /// @param _paused    The pause status of withdrawals from an chain in the superchain.
+    /// @param _sequencers The initial list of allowed sequencers
+    function initialize(
+        address _owner,
+        address _initiator,
+        address _vetoer,
+        address _guardian,
+        uint256 _delay,
+        bool _paused,
+        Types.SequencerKeys[] memory _sequencers
+    )
+        public
+        reinitializer(2)
+    {
+        __Ownable_init();
+        transferOwnership(_owner);
+
+        initiator = _initiator;
+        vetoer = _vetoer;
+        guardian = _guardian;
+        delay = _delay;
+        paused = false;
```

Comment:
This doesn't use the variable passed in but hardcodes `false`?

Reply:
fixed.